staging/kos/pkg/controllers/connectionagent/connectionagent.go

```diff
@@ -1126,14 +1126,14 @@ func (ca *ConnectionAgent) onRemoteAttAdded(obj interface{}) {
 	ca.queue.Add(attNSN)
 }
 
-func (ca *ConnectionAgent) onRemoteAttUpdated(oldObj, newObj interface{}) {
-	oldAtt := oldObj.(*netv1a1.NetworkAttachment)
-	newAtt := newObj.(*netv1a1.NetworkAttachment)
-	glog.V(5).Infof("Remote NetworkAttachments cache for VNI %06x: notified of update from %#+v to %#+v",
-		newAtt.Status.AddressVNI,
-		oldAtt,
-		newAtt)
-	ca.queue.Add(parse.AttNSN(newAtt))
+func (ca *ConnectionAgent) onRemoteAttUpdated(oldObj, obj interface{}) {
+	oldAtt, att := oldObj.(*netv1a1.NetworkAttachment), obj.(*netv1a1.NetworkAttachment)
+	glog.V(5).Infof("Remote NetworkAttachments cache for VNI %06x: notified of update from %#+v to %#+v.", att.Status.AddressVNI, oldAtt, att)
+	if oldAtt.Status.IPv4 != att.Status.IPv4 || oldAtt.Status.HostIP != att.Status.HostIP {
```

Comment:
For efficiency enqueue only if the update can lead to a change of the interface. This was possible even without immutability, but the if condition would have been longer.